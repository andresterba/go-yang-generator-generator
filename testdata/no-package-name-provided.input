---
generator_options:
  - option: output_file
    value: ./yang.go
models:
  - ../../../models/openconfig/release/models/local-routing/openconfig-local-routing.yang
  - ../../../models/openconfig/release/models/system/openconfig-messages.yang
path_to_generator: ../../../build-tools/generator
path_to_models: ../../..models/openconfig
