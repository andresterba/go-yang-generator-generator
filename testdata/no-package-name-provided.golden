package ocstructs

// This file is a placeholder in order to ensure that Go does not
// find this directory to contain an empty package.

//go:generate ../../../build-tools/generator -path=../../..models/openconfig  -output_file=./yang.go   ../../../models/openconfig/release/models/local-routing/openconfig-local-routing.yang  ../../../models/openconfig/release/models/system/openconfig-messages.yang 
